<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeComponent</class>
 <widget class="QWidget" name="MeComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="profilePicture">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="Bio_lbl">
     <property name="text">
      <string>Biography:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lastName_lbl">
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Sign up as a company</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="firstName_lbl">
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="ability_lbl">
     <property name="text">
      <string>Ability:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="editPicBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/edit.svg</normaloff>:/edit.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="delPicBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/trash.png</normaloff>:/trash.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
